roles/ceph-ansible/README.rst

```diff
@@ -0,0 +1,14 @@
+===================================
+ Setup ceph-ansible to deploy Ceph
+===================================
+
+This role setups Ceph-ansible to be able to install Ceph.
+Role can be configured by variables mentioned in ``defaults/main.yml``.
+As it is written in ``tasks/main.yml`` ceph-ansible package should be 
+preinstalled or available in enabled yum repository.
+
+Example usage of configured ceph-ansible:
```

Comment:
Following lines sounds slightly strangely (or maybe I just misunderstand them). Could you please check it and at least fix the format, so it will be properly generated? (check the View button)